Src/Sentry.Xamarin/Internals/NativeStackTraceFactory.ios.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Sentry.Extensibility;
+using Sentry.Protocol;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+
+namespace Sentry.Xamarin.Internals
+{
+    internal class NativeStackTraceFactory
+    {
+        private static string _nativeRegexFormat => "(?<id>\\d+)\\s+(?<method>[a-zA-Z\\.-_?]+)\\s+(?<offset>0x[0-9a-fA-F]+)\\s+(?<function>.+?(?=\\s\\+))\\s+\\+\\s+(?<line>\\d+)";
+        //"(?<id>\d+)\s+(?<method>[a-zA-Z\.-?]+)\s+(?<offset>0x[0-9a-fA-F]+)\s+(?<function>.+?(?=\s\+))\s+\+\s+(?<line>\d+)"
+        private readonly SentryXamarinOptions _options;
+
+        public NativeStackTraceFactory(SentryXamarinOptions options) => _options = options;
+
+        internal bool IsNativeException(string exceptionValue)
+            => exceptionValue.StartsWith("Objective-C exception");
+
+        internal SentryStackTrace CreateNativeStackTrace(SentryStackTrace managedStackTrace, IEnumerable<string> nativeStackTrace)
+        {
+            var nativeFramesList = new List<SentryStackFrame>();
+            for (int i = nativeStackTrace.Count() - 1; i >= 0; i--)
+            {
+                var match = Regex.Match(nativeStackTrace.ElementAt(i), _nativeRegexFormat);
+                if (match.Success)
+                {
+                    var method = match.Groups["method"].Value;
+                    nativeFramesList.Add(new SentryStackFrame()
+                    {
+                        Platform = "cocoa",
```

Comment:
I don't think `platform=cocoa` is valid here. Isn't this supposed to be `native?
```suggestion
                        Platform = "native",
```

Reply:
I'll change to native.
I was testing different names like cocoa and iOS (found it here https://github.com/getsentry/sentry-cocoa/blob/4e1b2fa5df554ddc9e30ca405b25fda89375e796/Tests/SentryTests/Protocol/TestData.swift#L122 )
but it seems like the result at the same output at Sentry UI.

Reply:
`native` can trigger symbolication, so if you have symbols, not sure if we upload it it'll work already or if it needs more data